### PR TITLE
Update Hydrawise for config flow

### PR DESCRIPTION
**Pull request by dknowles2**
## Proposed change
Update Hydrawise docs to reflect that it's now set up with config flow.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/95589
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist
- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by dknowles2** [comment]
> It seems that this PR is targeted against an incorrect branch. Documentation updates which apply to our current stable release should target the `current` branch. Please change the target branch of this PR to `current` and rebase if needed. If this is documentation for a new feature, please add a link to that PR in your description.

I think this is wrong? The new feature is based on the core `dev` branch, so updating docs in `current` would be incorrect?

**Comment by frenck** [comment]
> I think this is wrong? The new feature is based on the core `dev` branch, so updating docs in `current` would be incorrect?

It was correct, you missed adding the link to the Core PR, which you edited afterward:

CleanShot 2023-05-25 at 18 43 44

Because the parent PR was missing, it assumed it wasn't related to a code change (and thus targeting an incorrect branch).

That said, all is good now 👍 

../Frenck

File: source/_integrations/hydrawise.markdown [hunk 1]
```diff
@@ -26,114 +28,36 @@ There is currently support for the following device types within Home Assistant:
 - [Sensor](#sensor)
 - [Switch](#switch)
 
-## Configuration
+## Prerequisites
 
-To enable it, add the following to your `configuration.yaml` file:
+In order to set up the Hydrawise integration, you must first obtain an API Key.
```
Comment (frenck) [inline source/_integrations/hydrawise.markdown]:
```suggestion
To set up the Hydrawise integration, you must first obtain an API Key.
```

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by dknowles2** [comment]
> > I think this is wrong? The new feature is based on the core `dev` branch, so updating docs in `current` would be incorrect?
> 
> It was correct, you missed adding the link to the Core PR, which you edited afterward:
> 
> CleanShot 2023-05-25 at 18 43 44
> 
> Because the parent PR was missing, it assumed it wasn't related to a code change (and thus targeting an incorrect branch).
> 
> That said, all is good now 👍
> 
> ../Frenck

Ah, that makes sense! I wasn't sure which one to submit first since both PR templates request a reference to the other. 😵

**Pull request closed without merging**